### PR TITLE
Return statement: evaluate content for correct LoC markings

### PR DESCRIPTION
**Pull request by Slamdunk**
Fix https://github.com/sebastianbergmann/php-code-coverage/issues/907

### CODE REVIEW

File: tests/tests/Data/RawCodeCoverageDataTest.php [hunk 1]
```diff
@@ -362,6 +364,7 @@ public function testHeavyIndentationIsHandledCorrectly(): void
                 117,
                 120,
                 123,
+                125, // This shouldn't be marked as LoC, but it's high unlikely to happen IRL to have $var = []; on multiple lines
```
Comment (Slamdunk) [inline tests/tests/Data/RawCodeCoverageDataTest.php:367]: I've thoroughly checked each test that needed to be changed and everything is an edge case now fixed, except this one.

`$var = [];` on multiple lines will be marked on `[]` line too with this PR, which is wrong, but it's unlikely to happen on real code.

As far as I can tell, it's necessary to introduce this little bug to fix much more common ones.

**Comment by sebastianbergmann** [comment]
You are amazing, @Slamdunk. Thank you!

File: tests/tests/Data/RawCodeCoverageDataTest.php (hunk 1 above)
Comment (sebastianbergmann) [reply tests/tests/Data/RawCodeCoverageDataTest.php:367]: Makes sense to me.
